### PR TITLE
Update README.md to mention fudgeandrun

### PR DESCRIPTION
**Pull request by ronaldxs**
Describe use of fudgeandrun to simplify fudge and run/prove individual test files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,11 +111,14 @@ In cases where the current working directory makes a difference, the tests
 assume that the working directory is the root of the test suite, so that the
 relative path to itself is `t/spec/S\d\d-$section/$filename`.
 
-You can also run the `fudge` program manually. This command will fudge and run
-`prove` on a specific file:
-
-    ./fudge --keep-exit-code rakudo S05-metasyntax/angle-brackets.t &&
-    prove -eperl6 S05-metasyntax/angle-brackets.rakudo
+You can also run the `fudge` program manually. To fudge and run `prove` on a
+specific file:
```
Comment (zoffixznet) [inline README.md]: It says `run the fudge program manually` but the command runs `fudgeandrun`. The text probably needs to be updated?

Comment (ronaldxs) [inline README.md]: Looking at the commit log the sentence  

> You can also run the `fudge` program manually.

was added in the commit describing fudgeandrun for individual files, [d00fc0f](https://github.com/ronaldxs/roast/commit/d00fc0f99ea4b46f76dfd8c3f9836307ccbdc253#diff-04c6e90faac2675aa89e2176d2eec7d8).  I think that if the sentence is removed this PR works.  If you agree then I think I can update.  Thanks.

Comment (zoffixznet) [inline README.md]: Agreed

**Comment by zoffixznet** [comment]
Thanks.

**Pull request merged**